src/Raygun.NetCore.Blazor/Models/EnvironmentDetails.cs

```diff
@@ -39,19 +39,19 @@ internal record EnvironmentDetails
         /// The company who manufactured the browser.
         /// </summary>
         [JsonPropertyName("browser")]
-        public string BrowserManufacturer { get; set; }
+        public string? BrowserManufacturer { get; set; }
```

Comment:
Outside the scope of this PR, but in case it gets missed later, the JsonPropertyName meaning something different to the property name may be unintentional here.

Reply:
This was intentional. I was looking for API consistency between what the browser returned, to help end users get a clear understanding of what the data point actually was. The `JsonPropertyName` just mapped that clarity I was trying to provide at the code level to the value the Raygun API was expecting in the payload.